CONTRIBUTING.md

```diff
@@ -0,0 +1,190 @@
+enhancements, bug-fixes or test cases, we kindly ask you to follow this
+workflow:
+
+![Workflow Blockdiagram with Kroki](https://kroki.hzdr.de/blockdiag/svg/eNqNU8tO5DAQvPMVrTmRwwrYF7tCcNkT30A4dJyejDXGDm0bNkL8-7YfM3I0WsEpUXd1uV1VHoxT-1HjBG9nAHokhAeDAxm4hc1VB4uLDJZec2vzeJNA9oV80BMGarBfO1A7UnvQW9DeR4KL3s7RGGB6jjLQWzRMOC5Af7UPvpBtHe8blm9dqTDNzuvgeCmwgdGqXQP8LscJm6yAtVmA9bAG-eOAXK9T4DiODfRnlwvpzr1VzgbWQwza2QIOa-LrDoy2AdCOudXbLFY711uRF41Z4NzNqYCmK1xKN0y_OnjCPYGPLFuGHQb4cw8zek_-cK0XTa_EzdDvsuyhc-4rs4_TlPxx1rdWXnZinHI8O85alLkWXcafiKfW1ysJQa6dylcDk6YmdnHOEQI4js7iIjKtFNncZIxjTTZgqgjSiIRe4Uyl2Qbsy10JhHxrBuSv7pDQ7_87_MSLz5ycBBX-8BG5uJNcLxr6zzCL33nxaqP8N8ofzso6H3VNoNVb2zoz0riKuUByhJtWqPWUsKbcxkK61eYj4Pj4lTOO0x2NnnZhMJHa8K-7E9OySs1Jm_LTeT_7B8Nka6c=)
+
+For those of you who want to know how forked-based contributions work on
+_GitHub_, we would like to point to the _GitHub Docs_ about
+[contributing to projects](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
+
+## Set Up a Local Development Environment (Optional)
+
+In order to add contributions via pull requests you most probably need to
+create a local development environment to make sure that your changes
+work as expected.
+After cloning the project locally the _Python_ dependencies need to be
+installed:
+
+```shell
+$ pipenv install --dev
+```
+
+As mentioned in the suggested contribution workflow above, linting and testing
+your contributions locally is optional, but we encourage you to do so.
+The following sections show how to lint and test your contributions locally.
+
+### Linting the Project
+
+This role is linted via
+[Ansible Lint](https://ansible-lint.readthedocs.io/en/latest/)
+and
+[yamllint](https://yamllint.readthedocs.io/en/stable/)
+which guarantees that the role adheres to a set of
+[Ansible Lint rules](https://ansible-lint.readthedocs.io/en/latest/default_rules/)
+and
+[yamllint rules](https://yamllint.readthedocs.io/en/stable/rules.html):
+
+```shell
+$ pipenv run yamllint --strict --format colored .
+$ pipenv run ansible-lint -v --force-color --exclude .pipenv/ --offline . ./molecule
+```
+
+### REUSE Specification Compliance
+
+Each file in this project needs to have a proper license and copyright header.
+Please check that each file you add to the project contains license and 
+copyright information and that it thereby meets the
+[REUSE Specification](https://reuse.software/spec/):
+
+```shell
+$ pipenv run reuse lint
+```
+
+### Testing the Ansible Role
+
+This role is tested via
+[Ansible Molecule](https://molecule.readthedocs.io/en/latest/).
+During development of your changes we would like you to use _Molecule_ and
+the existing automated test cases to verify that your changes do not break
+existing test cases:
+
+```shell
+$ pipenv run molecule test
+````
+
+## How are Contributors Given Credit for Their Valuable Work?
+
+Please add yourself to the list of contributors in file 
+[README.md](README.md#contributors) 
```

Comment:
In case we would like to offer a contributors list in file README.md, we could add the following at the end of file README.md:

```
## Contributors

We would like to thank and give credits to the following contributors of this
project:

* Be the first to be named here!
```

Reply:
Thanks, @christianhueserhzdr, I added the contributors section to the README.